src/Lykke.Payments.Link4Pay/Lykke.Payments.Link4Pay.csproj

```diff
@@ -1,31 +1,39 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
```

Comment:
downgrade?

Reply:
it's not easy to use .net core 3.1 with current infrastructure packages